src/views/Settings.vue

```diff
@@ -88,14 +88,13 @@ export default defineComponent({
   methods: {
     setEComStore(store: any) {
       if(this.userProfile) {
-        this.store.dispatch('user/setEcomStore', {
-          'eComStore': this.userProfile.stores.find((str: any) => str.productStoreId == store['detail'].value)
-        })
+        const productStore = this.userProfile.stores.find((str: any) => str.productStoreId === store['detail'].value);
+        this.store.dispatch('user/setEcomStore', productStore ? productStore.productStoreId : {});
       }
     },
     setShopifyConfig(event: any){
-      const shopifyConfig = this.shopifyConfigs.find((shopifyConfig: any) => shopifyConfig.shopifyConfigId === event.detail.value)
-      this.store.dispatch('user/setCurrentShopifyConfig', shopifyConfig);
+      const currentShopifyConfig = this.shopifyConfigs.find((shopifyConfig: any) => shopifyConfig.shopifyConfigId === event.detail.value);
```

Comment:
Refer comments provided for the product store

Reply:
Sure, sure.